tests/foreman/api/test_http_proxy.py

```diff
@@ -293,7 +293,8 @@ def test_positive_sync_proxy_with_certificate(request, target_sat, module_org, m
     # Create and fetch new cerfiticate
     target_sat.custom_cert_generate(proxy_host)
     cacert = target_sat.execute(f'cat {cacert_path}').stdout
-    assert 'BEGIN CERTIFICATE' and 'END CERTIFICATE' in cacert
+    assert 'BEGIN CERTIFICATE'
+    assert 'END CERTIFICATE' in cacert
```

Comment:
Same issue as above.

Reply:
Same answer as above, with a slight modification. It now appears the test author is incorrect in their assumption of how this code works. Consider the following:
```python
>>> 1 and 2 in [2, 3, 4]
True
```

Reply:
Yeah there has to be double in operation or set operations to find the items in other list.

Reply:
for test failure clarity, i think having both as separate assertions would be better

Reply:
Yeah for now we can leave it as two separate operations and later Component owner can refactor if needed.

Reply:
Looking back on this, it seems unnecessary to check for both the beginning and end of the certificate, given this is an http_proxy test, it's assumed the generated cert would have beginning and end, no? Perhaps we could just make this a single assertion- 
`assert 'BEGIN CERTIFICATE' in cacert`